### PR TITLE
[WL-1641] Add new distribution metric for time to submit code

### PR DESCRIPTION
**Pull request by jdmunro**
This PR adds a new distrubution metric for measuring time to submit code from campus.

### CODE REVIEW

File: config/metricConfigs/learnMetrics.ts [hunk 1]
```diff
@@ -389,6 +389,13 @@ const allowedMetrics: MetricsConfig = [
     labels: [],
     protocol: "statsd"
   },
+  {
+    name: "campus__time_to_submit_code",
+    help: "measures the time to submit a code and get the response",
+    type: "distribution",
+    labels: [],
```
Comment (jdmunro) [inline config/metricConfigs/learnMetrics.ts:396]: The labels here don't seem to do anything, so skipped these for now. Feel free to disagree.

Comment (Smidge) [reply config/metricConfigs/learnMetrics.ts:396]: I agree that they don't seem to do anything other than document the intended fields

**Review by Smidge — APPROVED** [review]
So the main difference here is trying out distribution over histogram?

**Comment by jdmunro** [comment]
@Smidge Yes, exactly! Going to run them side-by-side and observe any differences rather than replace the existing one directly (until we know that's what we want).

**Pull request merged**